scripts/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+if [ ${TEST_STYLE} -eq 1 ]; then
```

Comment:
use TEST_TYPE=UT or TEST_TYPE=ST to make it easy understanding

Reply:
fixed